docs/running/wdl.rst

```diff
@@ -3,15 +3,121 @@
 WDL in Toil
 ***********
 
-Support is still in the alpha phase and should be able to handle basic wdl files.  See the specification below for more
-details.
+Toil has beta support for running WDL workflows, using the ``toil-wdl-runner``
+command.
 
-How to Run a WDL file in Toil
------------------------------
-Recommended best practice when running wdl files is to first use the Broad's wdltool for syntax validation and generating
-the needed json input file.  Full documentation can be found on the repository_, and a precompiled jar binary can be
+Running WDL with Toil
+---------------------
+
+You can run WDL workflows with ``toil-wdl-runner``. Currently,
+``toil-wdl-runner`` works by using MiniWDL_ to parse and interpret the WDL
+workflow, and has support for workflows in WDL 1.0 or later (which are required
+to declare a ``version`` and to use ``inputs`` and ``outputs`` sections).
+
+You can write workflows like this by following the `official WDL tutorials`_.
+
+When you reach the point of `executing your workflow`_, instead of running with
+Cromwell:
+
+``java -jar Cromwell.jar run myWorkflow.wdl --inputs myWorkflow_inputs.json``
+
+you can instead run with ``toil-wdl-runner``:
+
+``toil-wdl-runner myWorkflow.wdl --inputs myWorkflow_inputs.json``
+
+.. _`official WDL tutorials`: https://wdl-docs.readthedocs.io/en/stable/
+.. _`executing your workflow`: https://wdl-docs.readthedocs.io/en/stable/WDL/execute/
+
+This will default to executing on the current machine, with a job store in an
+automatically determined temporary location, but you can add a few Toil options
+to use other Toil-supported batch systems, such as Kubernetes:
+
+``toil-wdl-runner --jobStore aws:us-west-2:wdl-job-store --batchSystem kubernetes myWorkflow.wdl --inputs myWorkflow_inputs.json``
+
+For Toil, the ``--inputs`` is optiona, and inputs can be passed as a positional
+argument:
+
+``toil-wdl-runner myWorkflow.wdl myWorkflow_inputs.json``
+
+You can also run workflows from URLs. For example, to run the MiniWDL self test
+workflow, you can do:
+
+``toil-wdl-runner https://raw.githubusercontent.com/DataBiosphere/toil/36b54c45e8554ded5093bcdd03edb2f6b0d93887/src/toil/test/wdl/miniwdl_self_test/self_test.wdl https://raw.githubusercontent.com/DataBiosphere/toil/36b54c45e8554ded5093bcdd03edb2f6b0d93887/src/toil/test/wdl/miniwdl_self_test/inputs.json``
+
+Toil WDL Runner Options
+-----------------------
+
+**'-\\-jobStore'**: Specifies where to keep the Toil state information while
```

Comment:
This (and the options that follow) looks like it renders as `'-\-jobStore'` when I click "View File" in the review tab.  Is this a discrepancy with Sphinx?

Reply:
Read The Docs doesn't render the existing option list with with a slash in the middle: https://toil.readthedocs.io/en/latest/running/wdl.html#toilwdl-py-options

I'm not sure if we really need to have the slash though.